cppcache/integration-test/testThinClientPdxInstance.cpp

```diff
@@ -1907,19 +1886,16 @@ DUNIT_TASK_DEFINITION(CLIENT2, modifyPdxInstance)
     rptr->put(keyport1, wpiPtr);
     newPiPtr = std::dynamic_pointer_cast<PdxInstance>(rptr->get(keyport1));
     LOGINFO("ChildPdx object put get done");
-    ASSERT((*pIPtr.get() == *newPiPtr.get()) == false,
-           "PdxInstance should not be equal");
+    ASSERT((*pIPtr == *newPiPtr) == false, "PdxInstance should not be equal");
     childObjPtr = newPiPtr->getCacheableField("m_childPdx");
     ASSERT(childObjPtr != nullptr, "childObjPtr should not be nullptr");
     LOGINFO("got childPdx field ");
     auto cpi1 = std::dynamic_pointer_cast<PdxInstance>(childObjPtr);
-    ASSERT((*cpi.get() == *cpi1.get()) == false,
-           "PdxInstance should not be equal");
+    ASSERT((*cpi == *cpi1) == false, "PdxInstance should not be equal");
 
     auto cpo = std::dynamic_pointer_cast<ChildPdx>(cpi1->getObject());
     LOGINFO("got childPdx getObject ");
-    ASSERT((cpo.get()->equals(*childpdxobjPtr.get())) == true,
-           "child pdx should be equal");
+    ASSERT(cpo->equals(*childpdxobjPtr) == true, "child pdx should be equal");
```

Comment:
Same here, `== true` is redundant

Reply:
Again, this was mostly automated. I'd like to keep it that way.